src/pages/HomePage.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export default function HomePage() {
```

Comment:
Not important, but maybe try to use either function declarations or arrow functions regularly across the example.